newIDE/app/src/SceneEditor/index.js

```diff
@@ -1175,10 +1195,12 @@ export default class SceneEditor extends React.Component<Props, State> {
                   canUndo: () => canUndo(this.state.history),
                   canRedo: () => canRedo(this.state.history),
                   saveToHistory: () =>
+                    // TODO: pass actionType as argument of saveToHistory
```

Comment:
To remove?

Reply:
Or should I suggest to put a TODO at the beginning of the file: `History to review and we should ensure the context is well configured on each change`